mealy/error_visualizer.py

```diff
@@ -187,9 +187,11 @@ def plot_feature_distributions_on_leaves(self, leaf_selector='all_errors',
 
         Args:
 
-            leaf_selector (int or list or str): the desired leaf nodes to visualize. When int it represents the
-                number of the leaf node, when a list it represents a list of leaf nodes. When a string, the valid values
-                are either 'all_error' to plot all leaves of class 'Wrong prediction' or 'all' to plot all leaf nodes.
+            leaf_selector (None, int or array-like): the desired leaf nodes to visualize.
+                The leaves whose information will be returned
```

Comment:
Only onf of the sentences should be kept, either `the desired leaves to visualize` or `The leaves whose information would be returned`, as they are redundant. COnsidering we use `information` in the list of possible values, I'd suggest to keep the second one?

Same comment for error_analyzer on l. 311 and 495

Reply:
yes, second one. agree.